example/lib/main.dart

```diff
@@ -154,6 +202,36 @@ class _MyAppState extends State<MyApp> {
             ),
           ),
         ]),
+        Row(mainAxisAlignment: MainAxisAlignment.spaceEvenly, children: [
+          TextButton(
+              onPressed: () {
+                fetchClient();
+              },
+              key: const Key('initClient'),
+              child: Container(
+                color: Colors.lightBlue,
+                padding:
+                    const EdgeInsets.symmetric(vertical: 5, horizontal: 10),
+                child: const Text(
+                  'InitClient',
```

Comment:
we should rename this to fetchClient and the existing Init button should be getClient to be consistent with the public api (init is really an informal term)